forms/settings_form.py

```diff
@@ -0,0 +1,7 @@
+__author__ = 'Alex'
+from wtforms import Form, StringField, validators
+
+
+class SettingsForm(Form):
+    first_name = StringField('First Name', [validators.Length(min=4, max=25)])
```

Comment:
in https://trello.com/c/jfNFktsQ/15-manage-user-settings i actually wanted name and company_name, not first and last name :P